src/daemon/pex/pex.go

```diff
@@ -570,11 +624,13 @@ func backoffDownloadText(url string) (string, error) {
 	return body, nil
 }
 
-// parseRemotePeerList parses a remote peers.txt file
+// ParseRemotePeerList parses a remote peers.txt file
 // The peers list format is newline separated ip:port
 // Any lines that don't parse to an ip:port are skipped
 // Localhost ip:port addresses are ignored
-func parseRemotePeerList(body string) []string {
+//
+// FIXME : move to iputil
```

Comment:
It doesn't need to go into iputil since this is specific to pex

Reply:
Fixed